css/_variables.scss

```diff
@@ -126,4 +126,11 @@ $defaultDarkFontColor: #000;
 $inputControlEmColor: #f29424;
 //buttons
 $linkFontColor: #489afe;
-$linkHoverFontColor: #287ade;
\ No newline at end of file
+$linkHoverFontColor: #287ade;
+
+/**
+* Landing
+*/
```

Comment:
I already commented on a previous PR of yours about the above formatting of documentation comments but here it goes again: that's not the convention in the file and in the project.